src/sentry/static/sentry/app/views/onboarding/projectSetup/index.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import posed, {PoseGroup} from 'react-pose';
+import styled from 'react-emotion';
+
+import {t} from 'app/locale';
+import InlineSvg from 'app/components/inlineSvg';
+import Tooltip from 'app/components/tooltip';
+
+import InviteMembers from './inviteMembers';
+import LearnMore from './learnMore';
+import ProjectDocs from './projectDocs';
+import SetupChoices from './setupChoices';
+
+const ChoiceStar = styled(p => (
+  <Tooltip title={t('Recommended next step')}>
+    <InlineSvg src="icon-star-small-filled" {...p} />
+  </Tooltip>
+))`
+  position: absolute;
+  top: 10px;
+  right: 10px;
+  color: ${p => p.theme.yellow};
+`;
+
+const SETUP_CHOICES = [
+  {
+    id: 'install_guide',
```

Comment:
Are these `ids` used on server?

Reply:
no. They will likely be used fro analytics though.